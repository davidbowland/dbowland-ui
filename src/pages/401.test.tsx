import React from 'react'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import AuthorizationRequired from './401'

describe('401 error page', () => {
  test('Ensure snapshot match', () => {
    const { container } = render(<AuthorizationRequired />)
    expect(container.firstChild).toMatchSnapshot()
  })

  test('Ensure title is on the page', () => {
    const { getByText } = render(<AuthorizationRequired />)
    expect(getByText('401: Authorization Required')).toBeInTheDocument()
  })
})
