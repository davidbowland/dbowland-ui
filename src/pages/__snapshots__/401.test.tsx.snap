// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`401 error page Ensure snapshot match 1`] = `
<main
  class="sc-dlnjwi gnIxof"
>
  <h1
    class="sc-gtsrHT elXEts"
  >
    401: Authorization Required
  </h1>
  <p
    class="sc-bdnxRM fSObbe"
  >
    You have requested a resource that requires authorization. Please authenticate yourself before retrying your request.
    <br />
    <br />
    <a
      href="/"
    >
      Go home
    </a>
    .
  </p>
</main>
`;
